.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "modules": false
+    }]
```

Comment:
You probably want to define the exact targets here. See https://babeljs.io/docs/plugins/preset-env/.

Reply:
We support all browsers from this matrix https://github.com/getsentry/raven-js
Could you please update the config reflect it?